src/actions/info-core-version.ts

```diff
@@ -13,12 +15,19 @@ export class Action implements Actions.Action {
     @Container.inject(Identifiers.CLI)
     private readonly cli!: Cli.Application;
 
+    @Container.inject(Identifiers.Version)
+    private readonly currentPluginVersion!: string;
+
     public name = "info.coreVersion";
 
     public async execute(params: object): Promise<any> {
         return {
             currentVersion: this.app.version(),
+            installedVersion: this.getInstalledVersion(),
             latestVersion: await this.getLatestVersion(),
+            currentPluginVersion: this.currentPluginVersion,
```

Comment:
```suggestion
            currentManagerVersion: this.currentManagerVersion,
```

Same for all of the others where it says generic `Plugin` but is specifically talking about the `Manager` version. Ideally turn this into an object as `manager: { latest, current, installed }`.